scripts/fuzz/generate_decoding_fuzz_corpus.py

```
@@ -525,6 +525,34 @@ def main() -> int:
+        ROOT
+        / "docs/paper/artifacts/stwo-richer-gemma-window-family-v1-2026-04-22"
+        / "phase113-richer-gemma-window-family-w8.stwo.json"
+    )
+    phase113_path = (
+        corpus_root / "phase113_richer_gemma_window_family" / "valid_phase113.json"
+    )
+    phase113_path.parent.mkdir(parents=True, exist_ok=True)
+    if not phase113_source.exists():
+        raise FileNotFoundError(
+            f"missing phase113 fuzz-corpus source fixture: {phase113_source}"
+        )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)